docs/source/en/guides/collections.md

```diff
@@ -0,0 +1,187 @@
+>>> collection = get_collection("TheBloke/recent-models-64f9a55bb3115b4f513ec026")
+>>> collection
+Collection: { 
+  {'description': "Models I've recently quantized.',
+   'items': [...],
+   'last_updated': datetime.datetime(2023, 9, 21, 7, 26, 28, 57000, tzinfo=datetime.timezone.utc),
+   'owner': 'TheBloke',
+   'position': 1,
+   'private': False,
+   'slug': 'TheBloke/recent-models-64f9a55bb3115b4f513ec026',
+   'theme': 'green',
+   'title': 'Recent models'}
+}
+>>> collection.items[0]
+CollectionItem: { 
+  {'item_object_id': '6507f6d5423b46492ee1413e',
+   'author': 'TheBloke',
+   'item_id': 'TheBloke/TigerBot-70B-Chat-GPTQ',
+   'item_type': 'model',
+   'lastModified': '2023-09-19T12:55:21.000Z',
+   'position': 0,
+   'private': False,
+   'repoType': 'model'
+   (...)
+  }
+}
+```
+
+The [`Collection`] object returned by [`get_collection`] contains:
+- high-level metadata: `slug`, `owner`, `title`, `description`, etc.
+- a list of [`CollectionItem`] objects. Each item represents a model, a dataset, a Space or a paper.
+
+All items of a collection are guaranteed to have:
+- a unique `item_object_id`: this is the id of the collection item in the database
+- an `item_id`: this is the id on the Hub of the underlying item (model, dataset, Space, paper). It is not necessarily unique! only the `item_id`/`item_type` pair is unique.
+- an `item_type`: model, dataset, Space, paper.
+- the `position` of the item in the collection. Position can be updated to re-organize your collection (see [`update_collection_item`] below)
+
+A `note` can also be attached to the item. This is useful to add additional information about the item (e.g. a comment, a link to a blog post, etc.). If an item doesn't have a note, the attribute still exists with a `None` value.
+
+In addition to these base attributes, returned items can have additional attributes depending on their type: `author`, `private`, `lastModified`, `gated`, `title`, `likes`, `upvotes`, etc. None of these attributes are guaranteed to be returned.
+
+## Create a new collection
+
+Now that we know how to get a [`Collection`], let's create our own! Use [`create_collection`] with a title and optionally a description.
+
+```py
+>>> from huggingface_hub import create_collection
+
+>>> collection = create_collection(
+...     title="ICCV 2023",
+...     description="Portfolio of models, papers and demos I presented at ICCV 2023",
+... )
+```
+
+It will return a [`Collection`] object with the high-level metadata (title, description, owner, etc.) and an empty list of items. You will now be able to refer to this collection using it's `slug`.
+
+```py
+>>> collection.slug
+'iccv-2023-15e23b46cb98efca45'
+>>> collection.title
+"ICCV 2023"
+>>> collection.owner
+"username"
+```
+
+To create a collection on an organization page, pass `namespace="my-cool-org"` when creating the collection. Finally, you can also create private collections by passing `private=True`.
```

Comment:
I think this paragraph makes more sense if you put it at the beginning of the section when you're describing `create_collection`